SwrveSDK/src/main/AndroidManifest.xml

```diff
@@ -64,9 +61,8 @@
             </intent-filter>
         </receiver>
 
-        <service android:name="com.swrve.sdk.locationcampaigns.LocationCampaignFilterService"></service>
+        <service android:name="com.swrve.sdk.locationcampaigns.LocationCampaignFilter"/>
```

Comment:
I think I had both the old and new ones together in code, so the name change wasn't exactly on purpose.....